doc/development_guide/contributor_guide/governance.rst

```diff
@@ -0,0 +1,48 @@
+==============
+ Governance
+==============
+
+How to become a triager ?
+-------------------------
+
+Contribute for more than 3 releases consistently. Do not be too opinionated, follow the code of conduct without
+requiring emotional works from the maintainers. It does not mean that disagreements are impossible, only that arguments
+should stay technical and reasonable so the conversation is civil and productive. Get unanimous approval or neutral
+agreement from current maintainers.
+
+How to become a maintainer ?
+----------------------------
+
+Contribute two big code merge requests over multiple releases (for example like one checker in 2.13 and one
+complicated bug fixes in 2.14). Otherwise contributing for more than 3 releases consistently with
+great technical and interpersonal skills. Become an expert in a part of the code that is not maintained
+well at the moment (if we feel we can't merge something without your review, you're going to be able to
+merge those yourself soon). Get unanimous approval or neutral agreement from current maintainers.
```

Comment:
Maybe we could add that another maintainer has to suggest it? Similar to how cpython handles new core developer suggestions.

Additionally, this feels a bit wrong. Do X and you have a "right" to become a maintainer. Maybe do this and other `might` suggest you can become a maintainer. There should also be a note not to ask for it 😄

Reply:
I feel like the "do not ask for it" rule might prevent a valuable signal (the person want to become maintainer and is not contributing only to make pylint better). But it's also a lot less spam for admins/maintainers so I added something.